CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#2355](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2355))
 - AwsLambdaInstrumentor sets `cloud.account.id` span attribute
   ([#2367](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2367))
+- AwsLambdaInstrumentor sets `cloud.resource_id`, `faas.invocation_id` span attributes (breaking)
```

Comment:
Could you add a `### Breaking changes` section in the CHANGELOG so we can capture these semver changes altogether?

Reply:
Actually it is done in this PR

Reply:
Sure will do!

Reply:
@tammy-baylis-swi we have the breaking changes section in main, could you please move it there please?

Reply:
Great! Moved and improved wording in [50b446e](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2372/commits/50b446e23d132794934bbba0138aa4dc413c90c6)